### ISSUE TITLE
Stuck on loading screen when going into arena battle.

### ISSUE THREAD

**Issue by Linxhacks** [Prodigy-Hacking/ProdigyMathGameHacking#1335]
Every time I choose to go into arena on prodigy and select an opponent and continue I get stuck on the prodigy loading screen.

**Comment by Linxhacks** [comment]
I also have tried restarting the game many times, disabling the extension, etc.

**Comment by afkvido** [comment]
I'm pretty sure the arena is just immune to hacks.

**Comment by 1603191909** [comment]
fix battle crash?

**Comment by fingerprint53** [comment]
hello

**Comment by 1603191909** [comment]
> I'm pretty sure the arena is just immune to hacks.

Yeah, if you win battle hack, claim reward, (*shakes head*) it doesn't work. You must win yourself.

**Comment by 1603191909** [comment]
> I also have tried restarting the game many times, disabling the extension, etc.

May be your pets. Do fix battle crash.
